src/scenes/GameScene.ts

```diff
@@ -27,9 +41,6 @@ export default class GameScene extends Phaser.Scene {
       officeTilesetKey,
       'assets/3_Modern_Office_Shadowless/Modern_Office_Shadowless_16x16.png'
```

Comment:
Depending on the scene, we may not want to always load these (it will take extra memory)

Although we are good to keep it as it is for now